doc/docs/usage/shell-completion.md

```diff
@@ -0,0 +1,51 @@
+# Shell completion
+
+Ankaios supports command completion for the `ank` CLI in various shells.
+
+!!! note
+
+    For dynamic completion (workloads etc.) to work, the `ank` CLI must be configured via environment variables.
+    To use a non-default server URL, provide `ANK_SERVER_URL`.
+    Also provide either `ANK_INSECURE=true` or `ANK_CA_PEM`, `ANK_CRT_PEM` and `ANK_KEY_PEM`.
```

Comment:
This might get confusing at some point if the user overwrites with command line arguments. Wouldn't it be possible to to pass the arguments to the spawned process?

Reply:
I think that is currently not supported by clap_complete but I created a discussion item https://github.com/clap-rs/clap/discussions/5708 for that.

Reply:
Alright.